src/main.ts

```diff
@@ -1,8 +1,19 @@
 import { NestFactory } from '@nestjs/core';
 import { ApplicationModule } from './app/app.module';
+import { ValidationPipe } from './app/validators/validation.pipe';
 
 async function bootstrap() {
 	const app = await NestFactory.create(ApplicationModule);
+	app.useGlobalPipes(new ValidationPipe());
+	app.use(function(req, res, next) {
+		if (req.headers.origin === 'http://blendtec.test' || req.headers.origin === 'https://old.blendtec.com') {
```

Comment:
Move these domain values into your .env file and then whatever variable you add should be added in Beanstalk. I'm happy to help with that. otherwise, looks great.